pages/news.py

```diff
@@ -18,6 +18,16 @@ def news_items():
     
     # Below news items are licensed GFDL (from old freenetproject.org website)
     return [
+        NewsItem("20151025", _("2015-10-25 - Prerelease and Windows installer"),
+_("""
+The Windows installer has updated translations for French, Indonesian, and
+Simplified Chinese, and is newly translated into Bulgarian, Croatian,
+and Japanese.
+
+In addition, Freenet unstable testing prerelease 1471-pre2 is now available.
+For details on what it contains and how to test it see the
+[developer mailing list](https://emu.freenetproject.org/pipermail/devl/2015-October/038449.html).
```

Comment:
URLs are non-translatable and may change in the future, so please don't include them in the translation text. Please consider using [reference-style links](http://daringfireball.net/projects/markdown/syntax#link) (or alternatively Python format strings) instead.